projects/WTI-API/src/main/config/Logging.java

```diff
@@ -1,188 +1,31 @@
+ * This class provides Logging services for the WTI-API project.  Clients can obtain an instance of the WTI logger
+ * by invoking method {@link #getLogger()}, which returns a singleton {@link edu.csus.ecs.pc2.core.log.Log} object.
  * 
  * @author John Clevenger, PC^2 Development Team (pc2@ecs.csus.edu)
  *
  */
 public class Logging {
 
-	//the singleton Logging object (an instance of this class)
-	static private Logging logger = null;
+	//the singleton PC2 Log object 
+	static private Log logger = null;
 	
-	//the underlying Java logger
-	private java.util.logging.Logger javaLogger;
-	
-	//private constructor to enforce singleton (call static getLogger() to obtain an instance of this class)
-	private Logging() {	}
-	
-	//the console logger to be used if/when console logging is enabled
-	private ConsoleHandler consoleHandler;
-	
-	//flag to indicate whether console logging should be initially enabled
-	public static boolean enableConsoleLoggingAtStartup = false;
-	
-	/**
-	 * Record a log message in the log file at the "Info" level.
-	 * @param msg the log message to record
-	 */
-	public void logInfo(String msg) {
-		this.javaLogger.log(Level.INFO, msg);
-	}
-	
-	/**
-	 * Record a log message in the log file at the "SEVERE" (Error) level.
-	 * @param msg the log message to record
-	 */
-	public void logError(String msg) {
-		this.javaLogger.log(Level.SEVERE, msg);
-	}
-	
-	/**
-	 * Record a log message in the log file at the "WARNING" level.
-	 * @param msg the log message to record
-	 */
-	public void logWarning(String msg) {
-		this.javaLogger.log(Level.WARNING, msg);
-	}
-	
-	/**
-	 * By default, log messages are sent only to the Logger's <I>log file</i>; invoking this method
-	 * causes the Logger to begin sending log messages to the console as well as to the log file.
-	 */
-	public void enableConsoleLogging() {
-		if (consoleHandler==null) {
-			consoleHandler = new ConsoleHandler();
-			consoleHandler.setFormatter(new MyFormatter());
-		}
-		javaLogger.addHandler(consoleHandler);
-	}
-	
-	/**
-	 * Disables (turns off) logging to the console.  Console logging is off by default; see
-	 * {@link Logging#enableConsoleLogging} to turn logging on.
-	 * This method has no effect if console logging is off when the method is called.
-	 */
-	public void disableConsoleLogging() {
-		javaLogger.removeHandler(consoleHandler);
-	}
-	
-	/**
 	
 	/**
-	 * Returns a singleton instance of the {@link Logging} class. The returned Logging object uses
-	 * an underlying {@link java.util.logging.Logger} to do the actual logging.  
+	 * Returns a singleton instance of the {@link Log} class which can be used for logging WTI messages.  
 	 *  
-	 * @return a {@link Logging} object on which logging operations can be invoked
+	 * @return a {@link Log} object on which logging operations can be invoked
 	 */
-	static public Logging getLogger() {
+	static public Log getLogger() {
```

Comment:
I do not understand this "requested change".  The comment says "it would be better to have a second constructor..."  but there is no  constructor referenced in the changed code.  In fact, the subject class (source file) has NO constructors at all (other of course than the default Java "no argument" constructor).   Can you please explain more clear what change you think should be made to the PR?